seatunnel-dist/pom.xml

```diff
@@ -488,6 +489,12 @@
                 </dependency>
 
                 <!-- jdbc driver -->
+                <dependency>
+                    <groupId>com.alipay.oceanbase</groupId>
+                    <artifactId>oceanbase-client</artifactId>
+                    <version>${oceanbase.version}</version>
+                    <scope>provided</scope>
+                </dependency>
```

Comment:
Should not use the previous internal version, just keep it same with `seatunnel-connectors-v2/connector-jdbc/pom.xml`